includes/classes/Authentication/Passwords.php

```diff
@@ -17,6 +17,11 @@ class Passwords {
 
 	use Singleton;
 
+	/**
+	 * Stores the Have I Been Pwned API URL
+	 */
+	const API_URL = 'https://api.pwnedpasswords.com/range/';
```

Comment:
**suggestion:** Let's use a more descriptive name for this const. We might want to introduce additional APIs in the future.